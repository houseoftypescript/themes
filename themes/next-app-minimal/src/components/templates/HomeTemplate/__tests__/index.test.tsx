import { render } from '@testing-library/react';
import { HomeTemplate } from '../index';

describe('HomeTemplate', () => {
  it('to match snapshot', () => {
    const { container } = render(<HomeTemplate />);
    expect(container).toMatchSnapshot();
  });
});
