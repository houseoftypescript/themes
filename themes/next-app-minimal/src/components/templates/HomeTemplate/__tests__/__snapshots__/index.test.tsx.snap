// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTemplate to match snapshot 1`] = `
<div>
  <div
    class="text-gray-600 antialiased"
  >
    <div
      class="bg-gray-100"
    >
      <nav
        class="py-8"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
        >
          <div
            class="flex items-center justify-between"
          >
            <a
              class="flex items-center gap-2"
              href="/"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge text-primary-500 css-tzssek-MuiSvgIcon-root"
                data-testid="DashboardIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
                />
              </svg>
              <span
                class="text-2xl uppercase text-primary-500"
              >
                App
              </span>
            </a>
          </div>
        </div>
      </nav>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="py-16 md:py-32"
        >
          <header
            class="flex flex-col gap-8 text-center"
          >
            <h1
              class="whitespace-pre-line text-5xl font-bold leading-hero text-gray-900"
            >
              The modern landing page for
              <br />
              <span
                class="text-primary-500"
              >
                React developers
              </span>
            </h1>
            <div
              class="text-2xl"
            >
              The easiest way to build a React landing page in seconds.
            </div>
            <a
              href="#"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium px-6 py-4 font-extrabold css-12vt1u-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Download App
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </header>
        </div>
      </div>
    </div>
    <section
      class="py-8 md:py-16"
      id="features"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="mx-auto flex max-w-3xl flex-col gap-4 text-center"
        >
          <h1
            class="text-4xl font-bold text-gray-900"
          >
            Your title here
          </h1>
          <p
            class="text-xl md:px-20"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus malesuada nisi tellus, non imperdiet nisi tempor at.
          </p>
        </div>
        <div
          class="pt-8"
        >
          <div
            class="flex flex-wrap items-center md:flex-row-reverse"
          >
            <div
              class="w-full p-6 sm:w-1/2"
            >
              <img
                alt="First feature alt text"
                class="w-full"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="w-full text-center sm:w-1/2 md:text-left"
            >
              <h3
                class="text-3xl font-semibold text-gray-900"
              >
                Your title here
              </h3>
              <div
                class="mt-6 text-xl leading-9"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum, nunc non posuere consectetur, justo erat semper enim, non hendrerit dui odio id enim.
              </div>
            </div>
          </div>
          <div
            class="flex flex-wrap items-center "
          >
            <div
              class="w-full p-6 sm:w-1/2"
            >
              <img
                alt="Second feature alt text"
                class="w-full"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="w-full text-center sm:w-1/2 md:text-left"
            >
              <h3
                class="text-3xl font-semibold text-gray-900"
              >
                Your title here
              </h3>
              <div
                class="mt-6 text-xl leading-9"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum, nunc non posuere consectetur, justo erat semper enim, non hendrerit dui odio id enim.
              </div>
            </div>
          </div>
          <div
            class="flex flex-wrap items-center md:flex-row-reverse"
          >
            <div
              class="w-full p-6 sm:w-1/2"
            >
              <img
                alt="Third feature alt text"
                class="w-full"
                data-nimg="1"
                decoding="async"
                height="40"
                loading="lazy"
                src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                style="color: transparent;"
                width="40"
              />
            </div>
            <div
              class="w-full text-center sm:w-1/2 md:text-left"
            >
              <h3
                class="text-3xl font-semibold text-gray-900"
              >
                Your title here
              </h3>
              <div
                class="mt-6 text-xl leading-9"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse bibendum, nunc non posuere consectetur, justo erat semper enim, non hendrerit dui odio id enim.
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      class="py-8 md:py-16"
      id="cta"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="rounded-md bg-primary-100"
        >
          <div
            class="flex flex-col gap-8 p-8 text-center md:flex-row md:items-center md:justify-between md:text-left"
          >
            <div
              class="flex flex-col gap-2 text-2xl font-semibold"
            >
              <h1
                class="text-gray-900"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit.
              </h1>
              <h1
                class="text-primary-500"
              >
                Start your Free Trial.
              </h1>
            </div>
            <a
              href="#"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium px-6 py-4 font-extrabold css-12vt1u-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Download App
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
        </div>
      </div>
    </section>
    <footer
      class="bg-gray-100"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="py-8"
        >
          <div
            class="flex items-center justify-between"
          >
            <div>
              Â© 
              2023
               
              Nextjs Starter
            </div>
             
            <div
              class="flex items-center"
            >
              <a
                href="#"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium mx-2 text-gray-500 hover:text-gray-700 css-i4bv87-MuiSvgIcon-root"
                  data-testid="GitHubIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                  />
                </svg>
              </a>
              <a
                href="#"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium mx-2 text-gray-500 hover:text-gray-700 css-i4bv87-MuiSvgIcon-root"
                  data-testid="LinkedInIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                  />
                </svg>
              </a>
              <a
                href="#"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium mx-2 text-gray-500 hover:text-gray-700 css-i4bv87-MuiSvgIcon-root"
                  data-testid="TwitterIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
